### PR TITLE
Doc: Nano installation panel

### PR DESCRIPTION
**Pull request by liangs6212**
related issue: #6678
doc: https://liangs.readthedocs.io/en/nano-installaiton-panel/doc/Nano/Overview/install.html

WIP: Hope to find a way to replace if

### CODE REVIEW

**Comment by TheaperDeng** [comment]
1. it would be great if default version could be chosen automatically when tensorflow/pytorch is seletect
2. 2.70 -> 2.7 for tensorflow version

**Comment by liangs6212** [comment]
> 1. it would be great if default version could be chosen automatically when tensorflow/pytorch is seletect
> 2. 2.70 -> 2.7 for tensorflow version
> 3. if 1.13 and inference optimization yes are both selected, we need a second cmd for `neural_compressor`

Completed, but I haven't found a replacement for "if".

**Comment by jason-dai** [comment]
> 1. it would be great if default version could be chosen automatically when tensorflow/pytorch is seletect
> 2. 2.70 -> 2.7 for tensorflow version
> 3. if 1.13 and inference optimization yes are both selected, we need a second cmd for `neural_compressor`

Shall we upgrade to new TensroFlow and PyTorch versions as our default?

**Comment by TheaperDeng** [comment]
> > 1. it would be great if default version could be chosen automatically when tensorflow/pytorch is seletect
> > 2. 2.70 -> 2.7 for tensorflow version
> > 3. if 1.13 and inference optimization yes are both selected, we need a second cmd for `neural_compressor`
> 
> Shall we upgrade to new TensroFlow and PyTorch versions as our default?

We are working on upgrading tensorflow's default version.
We did not upgrade 1.12->1.13 for pytorch because of the delay release of ipex, Since now it has been released I think we could change the default version to 1.13.

**Review by TheaperDeng — APPROVED** [review]
LGTM

File: docs/readthedocs/source/doc/Nano/Overview/install.md [hunk 1]
```diff
@@ -40,4 +71,4 @@ In a conda environment, `source bigdl-nano-init` will also be added to `$CONDA_P
```
Comment (MeouSker77) [inline docs/readthedocs/source/doc/Nano/Overview/install.md:71]: The expression on line 70 is not very precise, `source bigdl-nano-init` won't be added to `$CONDA_PREFIX/etc/conda/activate.d/` until users run `source bigdl-nano-init` manually, We can point this out here or in the comments on line 64

Comment (MeouSker77) [reply docs/readthedocs/source/doc/Nano/Overview/install.md:71]: others LGTM

**Comment by liangs6212** [comment]
@TheaperDeng @rnwang04 The width of the `cmd` will be displayed adaptively according to the installation command.

File: docs/readthedocs/source/doc/Nano/Overview/install.md (hunk 1 above)
Comment (liangs6212) [reply docs/readthedocs/source/doc/Nano/Overview/install.md:71]: Done, please take a look at  some changes about cmd.

**Comment by rnwang04** [comment]
> @TheaperDeng @rnwang04 The width of the `cmd` will be displayed adaptively according to the installation command.

LGTM

**Comment by jason-dai** [comment]
Shall we show Inference Optimization "Yes" by default?

**Comment by liangs6212** [comment]
> Shall we show Inference Optimization "Yes" by default?

Done, users can use "inference" much earlier.

**Comment by jason-dai** [comment]
> > Shall we show Inference Optimization "Yes" by default?
> 
> Done, users can use "inference" much earlier.

Maybe switch the position of "yes" and "no"

**Comment by jason-dai** [comment]
LGTM; we do need to support TensorFlow versions beyond 2.7

**Pull request merged**